api/src/pcapi/core/mails/transactional/users/notification_before_deleting_suspended_account.py

```diff
@@ -0,0 +1,18 @@
+from pcapi.core import mails
+from pcapi.core.mails import models
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+import pcapi.core.users.models as users_models
+
+
+def get_send_email_before_deletion_of_suspended_account_data(user: users_models.User) -> models.TransactionalEmailData:
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏C'est une fonction locale au fichier
```suggestion
def _get_send_email_before_deletion_of_suspended_account_data(user: users_models.User) -> models.TransactionalEmailData: